actions/websocket_actions.jsx

```diff
@@ -175,11 +179,12 @@ export function initialize() {
 
 export function close() {
```

Comment:
Shouldn't all removals be on the handleClose instead of here? My reasoning is that if the websocket closes because of an error, it won't remove them, right?

Reply:
I think this is correct since we actually want to leave these listeners connected if the websocket disconnects randomly. The naming here is a bit confusing, but `initialize` is called when the `LoggedIn` component mounts and `close` is called when it unmounts, so I think adding listeners in one and removing them in the other makes sense